src/plugins/azgenerator/AzExtensionIncrementalGenerator.ts

```diff
@@ -40,6 +41,9 @@ export class AzExtensionIncrementalGenerator extends AzGeneratorBase {
         this.files[path.join(this.azDirectory, PathConstants.generatedFolder, PathConstants.initFile)] = GenerateNamespaceInit(this.model);
 
         this.files[path.join(this.azDirectory, PathConstants.testFolder, PathConstants.latestFolder, PathConstants.incTestScenarioFile(this.model.Extension_NameUnderscored))] = GenerateAzureCliTestScenario(this.model);
+        let testFile = PathConstants.testFolder + "/" + PathConstants.latestFolder + "/" + PathConstants.incTestScenarioFile(this.model.Extension_NameUnderscored);
+        this.files[path.join(this.azDirectory, testFile)] = inplaceGen(path.join(this.model.CliOutputFolder, this.azDirectory), testFile, GenerateAzureCliTestScenario(this.model));
```

Comment:
Consider to refactor GenerateAzureCliTestScenario to implement TemplateBase
And use generateIncrementalSingleAndAddtoOutput or generateFullSingleAndAddtoOutput to replace the dup logic here. current logic is dup in CoreFull CoreIncre ExtFull and ExtIncre generators.